src/test/javascript/portal/search/FacetedSearchResultsDataViewSpec.js

```diff
@@ -14,7 +14,7 @@ describe("Portal.search.FacetedSearchResultsDataView", function() {
     beforeEach(function() {
         facetedSearchDataView = new Portal.search.FacetedSearchResultsDataView({
             dataCollectionStore: {
-                isRecordActive: noOp,
+                getWithUuid: noOp,
```

Comment:
I thought you were going with `getByUuid`?  "with" doesn't really make sense with out the "dataCollection" bit before it.